gengokit/httptransport/httptransport.go

```diff
@@ -181,6 +180,31 @@ func createDecodeConvertFunc(f Field) string {
 	return fmt.Sprintf(fType, f.LocalName, f.LocalName+"Str")
 }
 
+// createDecodeTypeCast creates a go string that casts a 64 bit type to a 32 bit type
+// as strconv.ParseInt and ParseFloat always return the 64 bit type
+func createDecodeTypeCast(f Field) string {
```

Comment:
We only support the types in this table
https://developers.google.com/protocol-buffers/docs/proto3#scalar